_posts/2020-02-26-flink-for-data-warehouse.md

```diff
@@ -0,0 +1,107 @@
+
+If any of these resonate with you, you jsut found the right post to read: We have never been this close to the vision by strengthening Flink’s integration with Hive to a production grade.
+
+
+## Flink and Its Integration With Hive Comes into the Scene
+
+Apache Flink has been a proven scalable system to handle extremely high workload of streaming data in super low latency in many giant tech companies.
+
+Despite its huge success in real time processing domain, at its deep root, Flink has been faithfully following its inborn philosophy of being [an unified data processing engine for both batch and streaming](https://flink.apache.org/news/2019/02/13/unified-batch-streaming-blink.html), and taking a streaming-first approach in its architecture to do batch processing. By making batch a special case for streaming, Flink really leverages its cutting edge streaming capabilites and applies them to batch scenarios to gain the best offline performance. Flink’s batch performance has been quite outstanding in the early days, and has become even more impressive, as the community began to merge Blink, an Alibaba’s fork of Flink, back to Flink in 1.9 and finished it in 1.10, according to benchmarks we ran. 
+
+On the other hand, Apache Hive has established itself as a focal point of the data warehousing ecosystem. It serves as not only a SQL engine for big data analytics and ETL, but also a data management platform, where data is discovered and defined. As business evolves, it puts new requirements on data warehouse.
+
+Thus we started integrating Flink and Hive as a beta version in Flink 1.9. Over the past few months, we’ve been listening to users’ requests and feedback, further extensively enhancing our product, and running rigorous benchmarks (which will be published soon separately). I’m glad to announce that the integration between Flink and Hive is at production grade in [Flink 1.10](https://flink.apache.org/news/2020/02/11/release-1.10.0.html) right now and we can’t wait to walk you through the details.
+
+
+### Compatible with More Hive Versions
+
+The beta version in Flink 1.9 covers Hive 1.2 and 2.3, and rely on Hive’s own backward compatibility for other Hive versions.
+
+In Flink 1.10, we brought full coverage to all Hive versions including 1.0, 1.1, 1.2, 2.0, 2.1, 2.2, 2.3, and 3.1. Take a look [here](https://ci.apache.org/projects/flink/flink-docs-release-1.10/dev/table/hive/#supported-hive-versions).
```

Comment:
I'd prefer not to claim we support **"all"** Hive versions. We do miss some versions like 3.0.0